CHANGELOG.md

```diff
@@ -41,6 +41,7 @@
 * [#5019](https://github.com/bbatsov/rubocop/pull/5019): Fix auto-correct for `Style/HashSyntax` cop when hash is used as unspaced argument. ([@drenmi][])
 * [#5059](https://github.com/bbatsov/rubocop/issues/5059): Fix a false positive for `Style/MixinUsage` when `include` call is a method argument. ([@koic][])
 * [#5071](https://github.com/bbatsov/rubocop/pull/5071): Fix a false positive in `Lint/UnneededSplatExpansion`, when `Array.new` resides in an array literal. ([@akhramov][])
+* [#5039](https://github.com/bbatsov/rubocop/issues/5039): Fix `Rails/Delegate` by adding a necessary nil check. ([@asherkach][])
```

Comment:
No need for this since the code that introduced the regression is not released yet. 🙂

Reply:
Removed.